### PR TITLE
update(plugins/cloudtrail): Add a region setting.

### PR DESCRIPTION
**Pull request by geraldcombs**
Add a "region" configuration option which lets the user specify an AWS region.

The jsonschema module was truncating descriptions with commas, so remove them for now.

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area plugins

> /area registry

> /area build

> /area documentation

**What this PR does / why we need it**:

A completely unscientific survey of GitHub's search box returned the following:

AWS_PROFILE: 168K code hits, 124 wiki hits  
AWS_DEFAULT_PROFILE: 16K code hits, 13 wiki hits  
AWS_REGION: 643K code hits, 359 wiki hits  
AWS_DEFAULT_REGION: 2M code hits, 194 wiki hits  

Explicitly specifying a region seems to be a common use case, so we should probably handle it in the cloudtrail plugin.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by jasondellaluce** [comment]
Hey @geraldcombs, thanks for this! Can you sign-off your commit?

**Comment by geraldcombs** [comment]
> Hey @geraldcombs, thanks for this! Can you sign-off your commit?

Oops. Done.

File: plugins/cloudtrail/pkg/cloudtrail/aws.go [hunk 1]
```diff
@@ -48,6 +50,10 @@ func (p *PluginConfigAWS) ConfigAWS() (aws.Config, error) {
 		opts = append(opts, config.WithSharedConfigProfile(p.Profile))
 	}
 
+	if len(p.Region) > 0 {
```
Comment (Issif) [inline plugins/cloudtrail/pkg/cloudtrail/aws.go:53]: it's a string, I don't understand why you check the length and not just the emptiness `if p.Region != ""` (same question for `p.Config)`

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**